test/sunburst-ariaDecal2.html

```diff
@@ -0,0 +1,186 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+<html>
+    <head>
+        <meta charset="utf-8" />
+        <script src="lib/simpleRequire.js"></script>
+        <script src="lib/config.js"></script>
+    </head>
+    <body>
+        <style>
+            html,
+            body,
+            #main {
+                width: 100%;
+                height: 100%;
+                /* background-image: url(old_mathematics.png); */
+                /* background-repeat: repeat; */
+            }
+        </style>
+        <div id="main"></div>
+        <script>
+            var chart;

```

Comment:
The test result of this case is not correct. Since "Grandpa", "Nancy" and "Mike" have different colors, they should also have different decals, otherwise, decal is not useful at all.
Besides, this case can be put into `sunburst-ariaDecal.html` rather than in a seperate file.